avocado/core/output.py

```diff
@@ -639,6 +639,8 @@ def _log_line(self, line):
         Passes lines of output to the logging module.
         """
         for lg in self._logger:
+            if not lg.handlers:
+                lg.handlers.append(NULL_HANDLER())
```

Comment:
Btw I tried `open("/tmp/log", "aw").write(log.name + "\n")` on my machine every-time it's appended and it was actually never used, no matter whether I interrupted the execution before or after `_start_logging`.

When I check the missing handler during LogFile init, it added `paramiko` handlers. So I'm in doubt whether is this really used. Anyway I'm fine with having it here if we use the `continue` instead of `NULL_HANDLER`, because that is what it basically does.